silq/parameters/acquisition_parameters.py

```diff
@@ -329,6 +329,7 @@ def analyse(self,
 
     def plot_traces(self, filter=None, channels=['output'],
                     t_skip: Union[bool, float] = True,
+                    clim=None,
```

Comment:
general question: what is clim?

Reply:
haha you forced me to write documentation

Reply:
I see, all good now